hugo/layouts/partials/langium-mobile-menu.html

```diff
@@ -3,21 +3,18 @@
 x-transition:enter-start="opacity-0 scale-95" x-transition:enter-end="opacity-100 scale-100"
 x-transition:leave="transition ease-in duration-100 transform"
 x-transition:leave-start="opacity-100 scale-100" x-transition:leave-end="opacity-0 scale-95"
-class="absolute top-0 inset-x-0 p-2 transform origin-top-right md:hidden">
+class="absolute inset-x-0 top-0 p-2 origin-top-right transform md:hidden">
```

Comment:
Is there a particular reason why the css classes have been reordered?

Reply:
Good point! No it happens automatically on save...

Reply:
I disabled my Headwind extension. It was once required for another customer.